concent_api/middleman/decorators.py

```diff
@@ -0,0 +1,15 @@
+from functools import wraps
+
+
+def safe_execution(logger):
+    def wrapper(coroutine):
```

Comment:
Could you add a comment what this decorator does ?

Reply:
sure

Reply:
done